src/ActionList.jsx

```diff
@@ -19,6 +20,31 @@ export default class ActionList extends Component {
 
   componentDidMount() {
     this.scrollToBottom(true);
+    if (!this.props.draggableActions) return;
+    const container = ReactDOM.findDOMNode(this.refs.rows);
```

Comment:
I think it's better to use callback ref instead of string (see https://github.com/bevacqua/react-dragula#example-using-refs-es2015-syntax), as string refs are not recommended recently.